### PR TITLE
adding test for staff_constraint.rb

### PR DESCRIPTION
**Pull request by jethroo**
... to increase overall test coverage a bit ;)

### CODE REVIEW

**Comment by tarebyte** [comment]
Looks great thanks @jethroo!

File: spec/lib/staff_constraint_spec.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+describe StaffConstraint do
+  subject { described_class.new }
+
+  describe '#matches?' do
+    context 'when without user session' do
+      let(:request) { double('request', session: {}) }
+
+      it { expect(subject.matches?(request)).to be_falsey }
```
Comment (johndbritton) [inline spec/lib/staff_constraint_spec.rb]: The preference in this project is to use the more verbose `do...end` syntax for `it` blocks. The extra information provided makes the test output look really nice and easy to parse.

```ruby
it 'does not match the constraint' do
  expect(subject.matches?(request)).to be_falsey
end
```

Can you update this and the other `it` blocks to use this format?

Comment (johndbritton) [inline spec/lib/staff_constraint_spec.rb]: Just noticed this was already merged. If you're feeling extra nice, bonus :sparkles: for a follow up PR.

Comment (jethroo) [inline spec/lib/staff_constraint_spec.rb]: sure will do ;)

**Pull request merged**